azure-pipelines/powershell/Copy-SignedBits.ps1

```diff
@@ -0,0 +1,51 @@
+[cmdletbinding()]
```

Comment:
Talked to @bangbingsyb , we want to investigate how to remove this script and just have msbuild publish things in the right place